CMakeLists.txt

```diff
@@ -158,10 +158,12 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/orionTypes)
   ADD_SUBDIRECTORY(src/lib/ngsi)
   ADD_SUBDIRECTORY(src/lib/serviceRoutines)
+  ADD_SUBDIRECTORY(src/lib/serviceRoutinesV2)
```

Comment:
Any change related with this has been deferred. An issue has been created for this: https://github.com/telefonicaid/fiware-orion/issues/959